sass/_InitializationPlaceholder.scss

```diff
@@ -97,7 +97,8 @@
   @include placeholderLoading();
 }
 
-#{$allFacetsType} {
+#{$allFacetsType},
```

Comment:
Why not add it to the ${allFacetsType} variable in sass ?

Reply:
There's other styles that will get applied that I don't want.
Maybe later when I'll change the older facet styling I'll add the DynamicFacet to that variable, or make two variables one without DynamicFacet and one with really all of them

Reply:
Ok ... :/